opentelemetry-proto/src/opentelemetry/proto/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "1.4.0.dev0"
+__version__ = "1.4.0"
```

Comment:
Do we release the new version for the packages that didn't have any changes?

Reply:
Correct, before this release, we pinned all the dependencies to the latest released version, which had the side effect of forcing us to release every package every time. This will no longer be needed after 1.4.0

Reply:
Yes, we do. I personally disagree with doing so, but we do. :shrug: